### PR TITLE
Add Manufacturer and change some Fields in Place as blank=True

### PR DESCRIPTION
**Pull request by Vonuki**

### CODE REVIEW

File: benteler/models.py [hunk 1]
```diff
@@ -88,14 +98,15 @@ class Meta:
...
     active = models.BooleanField(default=True, verbose_name=_('Active'))
-    cost_center = models.ForeignKey(CostCenter, on_delete=models.CASCADE, verbose_name=_('CostCenter'))
+    cost_center = models.ForeignKey(CostCenter, on_delete=models.CASCADE, verbose_name=_('CostCenter'), blank=True, null=True )
     parts = models.ManyToManyField(Part, verbose_name=_('Parts'), blank=True)
     parent_place = models.ForeignKey('Place', blank=True, null=True, verbose_name=_('ParentPlace'))
 
     def __str__(self):
-        return self.name
+        #return self.name
+        return "{}-{}".format(self.parent_place.name, self.name)
```
Comment (ggevorg) [inline benteler/models.py]: parent_place may be None

File: benteler/models.py [hunk 2]
```diff
@@ -108,8 +119,9 @@ class WorkType(models.Model):
     parent = models.ForeignKey('WorkType', blank=True, null=True, verbose_name=_('ParentWorkType'))
 
     def __str__(self):
-        return self.name
-
+        #return self.name
+        return "{}-{}".format(self.parent.name, self.name)
```
Comment (ggevorg) [inline benteler/models.py]: parent may be None

**Review by ggevorg — COMMENTED** [review]
Need check on None value in methods __str__

**Review by aborilov — CHANGES_REQUESTED** [review]
pls fix

File: benteler/models.py [hunk 3]
```diff
@@ -88,14 +98,15 @@ class Meta:
 @python_2_unicode_compatible
 class Place(models.Model):
     name = models.CharField(max_length=255, verbose_name=_('Name'))
-    serial_number = models.CharField(max_length=255, verbose_name=_('SerialNumber'))
+    serial_number = models.CharField(max_length=255, verbose_name=_('SerialNumber'), blank=True)
     active = models.BooleanField(default=True, verbose_name=_('Active'))
-    cost_center = models.ForeignKey(CostCenter, on_delete=models.CASCADE, verbose_name=_('CostCenter'))
+    cost_center = models.ForeignKey(CostCenter, on_delete=models.CASCADE, verbose_name=_('CostCenter'), blank=True, null=True )
```
Comment (aborilov) [inline benteler/models.py]: line too long. 80 chars max

**Review by aborilov — CHANGES_REQUESTED** [review]

File: benteler/models.py [hunk 4]
```diff
@@ -100,13 +100,18 @@ class Place(models.Model):
     name = models.CharField(max_length=255, verbose_name=_('Name'))
     serial_number = models.CharField(max_length=255, verbose_name=_('SerialNumber'), blank=True)
     active = models.BooleanField(default=True, verbose_name=_('Active'))
-    cost_center = models.ForeignKey(CostCenter, on_delete=models.CASCADE, verbose_name=_('CostCenter'), blank=True, null=True )
+    cost_center = models.ForeignKey(CostCenter, on_delete=models.CASCADE, verbose_name=_('CostCenter'), 
+                                    blank=True, null=True )
```
Comment (aborilov) [inline benteler/models.py:104]: no need for space before the bracket

**Comment by aborilov** [comment]
@Vonuki you should get rid of those extra "Merge branch..." commit.
you should use rebase instead of merge

File: benteler/models.py [hunk 5]
```diff
@@ -88,14 +98,20 @@ class Meta:
...
     parts = models.ManyToManyField(Part, verbose_name=_('Parts'), blank=True)
     parent_place = models.ForeignKey('Place', blank=True, null=True, verbose_name=_('ParentPlace'))
 
     def __str__(self):
-        return self.name
+        #return self.name
+        if self.parent_place is None:
+            return "{}".format(self.name)
+        else:
+            return "{}-{}".format(self.parent_place.name, self.name)
```
Comment (aborilov) [inline benteler/models.py]: why do we need this at all?
where can such name be helpful?
I don't like it

File: benteler/models.py [hunk 6]
```diff
@@ -120,7 +125,10 @@ class WorkType(models.Model):
 
     def __str__(self):
         #return self.name
-        return "{}-{}".format(self.parent.name, self.name)
+        if self.parent.name is None:
```
Comment (aborilov) [inline benteler/models.py]: it won't work anyway, because `None` is `parent` not it's `name` field

File: benteler/models.py [hunk 7]
```diff
@@ -213,4 +224,19 @@ def __str__(self):
...
-        verbose_name_plural = _('Events')
\ No newline at end of file
+        verbose_name_plural = _('Events')
+
+@python_2_unicode_compatible        
+class Report(models.Model):
+    event = models.ForeignKey(Event, on_delete=models.CASCADE, blank=False, null=False, verbose_name=_('Event'))
+    reporter = models.ForeignKey(User, on_delete=models.CASCADE, blank=False, null=False, verbose_name=_('Reporter'))
+    date = models.DateField(default=datetime.today, blank=False, null=False, verbose_name=_('Date'))
+    worktime = models.IntegerField(verbose_name=_('WorkTime'))
```
Comment (aborilov) [inline benteler/models.py:234]: Why int? what is really mean?

Comment (aborilov) [inline benteler/models.py:234]: I think `DurationField` much better suit for this purpose.

**Comment by aborilov** [comment]
too many changes needed, we understand what we need to change and will do it by self

**Pull request closed without merging**